config/crd/bases/infrastructure.cluster.x-k8s.io_haproxyloadbalancers.yaml

```diff
@@ -231,6 +231,10 @@ spec:
                       value in the template from which the virtual machine is cloned.
                     format: int32
                     type: integer
+                  os:
```

Comment:
why is this needed for haproxy ? wouldnt it always be linux ? or just a boilerplate/generatino thing?

Reply:
just the auto-generation at work